static/app/views/projectInstall/platform.tsx

```diff
@@ -120,10 +131,81 @@ export function ProjectInstallPlatform({location, params, route, router}: Props)
     ? projects.find(proj => proj.slug === params.projectId)
     : undefined;
 
+  const {
+    data: projectAlertRules,
+    isLoading: projectAlertRulesIsLoading,
+    isError: projectAlertRulesIsError,
+  } = useApiQuery<IssueAlertRule[]>(
+    [`/projects/${organization.slug}/${project?.slug}/rules/`],
```

Comment:
when creating a project, an issue rule can be created in step 2, and the data is not saved in the project but in another location through this endpoint. That's why if we want to get those values again to fill out fields by default, we have to fire this request